diff --git a/language.py b/language.py
index 331c572..c632a02 100644
--- a/language.py
+++ b/language.py
@@ -124,7 +124,7 @@ PYTHON = Language(
     display_name="Python",
     extension="py",
     submission_lang_pattern=re.compile(
-        ".*Python \\(3.*|.*Python \\(CPython 3.*"),
+        ".*Python \\(PyPy 3.*"),
     default_code_generator=python.main,
     default_template_path=get_default_template_path('py'),
     compile_command="python3 -mpy_compile {filename}.py",
